backends/tfhe-cuda-backend/cuda/tests_and_benchmarks/include/setup_and_teardown.h

```diff
@@ -31,7 +31,7 @@ void programmable_bootstrap_multibit_setup(
     uint64_t **d_lut_pbs_identity, uint64_t **d_lut_pbs_indexes,
     uint64_t **d_lwe_ct_in_array, uint64_t **d_lwe_input_indexes,
     uint64_t **d_lwe_ct_out_array, uint64_t **d_lwe_output_indexes,
-    int8_t **pbs_buffer, int lwe_dimension, int glwe_dimension,
+   int lwe_dimension, int glwe_dimension,
```

Comment:
Just a small format issue here I believe.

Reply:
I was wondering with for fmt didn't catch this. It was not targeting `tests_and_benchmarks/include`. Thanks for that! It is already fixed (as well the formatting issue).